### PR TITLE
update block path

### PR DESCRIPTION
**Pull request by YevenLourance**
update block path

### CODE REVIEW

File: Source/moja.modules.gdal/src/writevariablegeotiff.cpp [hunk 1]
```diff
@@ -463,8 +463,9 @@ void WriteVariableGeotiff::DataSettingsT<T>::doLocalDomainProcessingUnitShutdown
    typename std::unordered_map<int, std::vector<T>>::iterator itPrev;
    for (auto it = _data.begin(); it != _data.end(); ++it) {
       const auto& [step, data] = (*it);
+      std::string block_path;
```
Comment (sea001) [inline Source/moja.modules.gdal/src/writevariablegeotiff.cpp:467]: do the declaration & assignment in 1 line.  can then also be made const

File: Source/moja.modules.gdal/src/writevariablegeotiff.cpp [hunk 2]
```diff
@@ -510,7 +511,7 @@ void WriteVariableGeotiff::DataSettingsT<T>::doLocalDomainProcessingUnitShutdown
                                    cellRows, gdal_type(_dataType), 0, 0);
          if (err != CE_None) {
             std::ostringstream oss;
-            oss << "Could not write to raster file: " << block_path.string() << std::endl;
+            oss << "Could not write to raster file: " << block_path<< std::endl;
```
Comment (sea001) [inline Source/moja.modules.gdal/src/writevariablegeotiff.cpp:514]: it's a shame this repeated code can't be pulled out of the if-else and done once

**Review by sea001 — COMMENTED** [review]
just two things

File: Source/moja.modules.gdal/src/writevariablegeotiff.cpp (hunk 1 above)
Comment (YevenLourance) [reply Source/moja.modules.gdal/src/writevariablegeotiff.cpp:465]: good point. changed

File: Source/moja.modules.gdal/src/writevariablegeotiff.cpp (hunk 2 above)
Comment (YevenLourance) [reply Source/moja.modules.gdal/src/writevariablegeotiff.cpp:512]: true, but not sure the details of rasterio. might keep it now and see if make these exceptions more detailed in different conditions or make them into one later on?

**Review by sea001 — APPROVED** [review]

**Pull request merged**